### PR TITLE
Remove OSS team from CODEOWNERS file and add backend team

### PR DESCRIPTION
**Pull request by mstruve**
## What type of PR is this? (check all applicable)
- [x] Documentation Update

## Description
Since we no longer have a dedicated OSS team and will be working off of our rotation I am removing the OSS team from Github. This means we will no longer use it via CODEOWNERS so I have removed it and replaced it in a couple of places with the Backend team. 

I thought about trying to update the team every week with the new people on the rotation but that seemed like it would be too much manual work and that I would eventually start missing or forget so I decided to remove it instead. Let me know if anyone else has thoughts or suggestions regarding this! 

alt_text

### CODE REVIEW

**Review by atsmith813 — APPROVED** [review]

**Review by citizen428 — APPROVED** [review]

**Comment by citizen428** [comment]
> Let me know if anyone else has thoughts or suggestions regarding this!

I think this is fine. I don't feel with our new structure the team is useful enough to spend any time on trying to automate this.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -9,20 +9,20 @@
 /app/services/search/                               @forem/sre
 /app/workers/                                       @forem/sre
 /config/                                            @forem/sre
-/db/                                                @forem/sre
+/db/                                                @forem/sre @forem/backend
 /docs/                                              @forem/oss    @jacobherrington
```
Comment (rhymes) [inline .github/CODEOWNERS:13]: missed one :)

**Review by rhymes — APPROVED** [review]
Agreed! Left one note

File: .github/CODEOWNERS (hunk 1 above)
Comment (mstruve) [inline .github/CODEOWNERS:13]:
```suggestion
/docs/                                              @jacobherrington
```

Comment (mstruve) [reply .github/CODEOWNERS]: hiding in plain sight 😂

**Pull request merged**